pennylane/wires.py

```diff
@@ -141,37 +123,23 @@ def __radd__(self, other):
         Returns:
             Wires: all wires appearing in either object
         """
-        other = Wires(other)
-        return Wires(Wires.all_wires([other, self]))
-
-    def __array__(self):
```

Comment:
Curious how come this was removed? This method allows `np.array(wires)` to work, which might be used by some of the plugins (but not 100% sure)

Reply:
`np.array(wires)` still works. I just tested it and did not go into the source of orderedset or numpy to see via which builtin it works. Essentially, `Wires` instances now only are a "special" list, and in particular its iteration behaviour is the same as for a list, so numpy will know what to do. Due to `Wires` instances being flat always, I think this should always behave as expected.